src/main/java/core/basesyntax/dao/UserDao.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public interface UserDao {
+    boolean addUser(User user);
+
+    User getUserById(long id);
+
+    List<User> getAllUsers();
+
+    void changeUserPassword(long id, String newPassword);
+
+    List<Bet> getAllUserBets(long id);
+
+    boolean addBetToUser(User user, Bet bet);
```

Comment:
I don't think these methods suit here

we can use `getUserById(long id)` and then do whatever we want

`addBetToUser` has no connection to storage at all, dao layer should only contain simple methods that directly modify storage